tests/smoke/render/render-reveal.test.ts

```diff
@@ -5,7 +5,25 @@
 *
 */
 
-import { docs } from "../../utils.ts";
+import { docs, outputForInput  } from "../../utils.ts";
+import { ensureHtmlElements } from "../../verify.ts";
 import { testRender } from "./render.ts";
 
+// demo file renders ok
 testRender(docs("reveal/index.qmd"), "revealjs", false);
+
+// aside and footnotes correctly moved with speaker notes ignored
+const input = docs("reveal/aside-footnotes.qmd");
+const output = outputForInput(input, "revealjs");
+testRender(input, "revealjs", false, [
+  ensureHtmlElements(output.outputPath, [
+    // speaker notes are left not moved
+    "section#slide-with-speaker-notes > aside.notes:last-child",
+    // div of class aside are moved
+    "section#slide-with-footnotes > aside:last-child > div > p",
+    // footnotes are put in aside
+    "section#slide-with-footnotes > aside:last-child > ol.aside-footnotes > li",
+    // footnotes back are removed
+    "section#slide-with-footnotes > aside:last-child > ol.aside-footnotes > li > :not(.footnote-back)",
```

Comment:
Just FYI if you're trying to validate the absence of elements in the DOM (for example, there are no `.footnote-back` elements), you can also pass an array of no-match selectors after the match selectors (the verification will check that those selectors do not return elements). 

```
 ensureHtmlElements(output.outputPath, [
     // speaker notes are left not moved
     "section#slide-with-speaker-notes > aside.notes:last-child",
     // div of class aside are moved
     "section#slide-with-footnotes > aside:last-child > div > p",
     // footnotes are put in aside
     "section#slide-with-footnotes > aside:last-child > ol.aside-footnotes > li"],
     [     
       // footnotes back are removed
       "section#slide-with-footnotes > aside:last-child > ol.aside-footnotes > li > .footnote-back"])
```